nemo/core/nemo_model.py

```diff
@@ -147,8 +159,7 @@ def __make_nemo_file_from_folder(filename, source_dir):
                             output=path.join(tmp_folder, module_checkpoint),
                             d_format=DeploymentFormat.TRTONNX,
                         )
-                    except Exception as ex:
-                        print(ex)
+                    except:
```

Comment:
Please do not use a bare except.
I find it very strange that you have a try, except block inside another try, except block

Reply:
fixed bare Except.  
The reason why there are two Excepts is actually dictated by why we are doing ".nemo" in the first place.
In case all modules can be converted to onnx, the inner one will not be triggered. But when something isn't convertable, the innermost except will trigger and save to .pt instead